components-compose/src/main/java/uk/gov/hmrc/components/compose/ui/theme/Theme.kt

```diff
@@ -271,5 +273,22 @@ fun ProvideHmrcTheme(
     )
 }
 
+@Composable
+fun textInputViewColors(): TextFieldColors = OutlinedTextFieldDefaults.colors(
```

Comment:
Composable functions should be written in Pascal case like: TextInputViewColors,